src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -1048,7 +1052,8 @@ public void stopTerminal() {
         } else if (getBuildCategory() == BuildType.GRADLE_TYPE) {
             keyboard.enterText("./gradlew libertyStop");
         } else {
-            TestUtils.printTrace(TestUtils.TraceSevLevel.INFO,  "Invalid build type specified");
+            LOGGER.warn("Invalid build type specified.");
```

Comment:
Could you provide a demo of what this looks like? I suspect it goes into a different file. What happens if you use trace level ERROR? Is it just display or does it throw an exception?

Reply:
When using `LOGGER.warn`, the message appears as a warning in the log rather than as info, unlike `TestUtils.printTrace`. I have added a SS below.

Reply:
I think it is important to stick to just one way of printing the trace. One day we may need to parse the output with a tool and we'll regret introducing another way. Could you change the lines that use LOGGER and use `TestUtils.printTrace()` with type Error? Also remove the import.

Reply:
Ok @turkeylurkey . I will change that